windows/DateTimePickerWindows/DateTimePickerView.idl

```diff
@@ -8,4 +8,10 @@ namespace DateTimePicker {
         DateTimePickerView(Microsoft.ReactNative.IReactContext context);
         void UpdateProperties(Microsoft.ReactNative.IJSValueReader reader);
     };
+
+    [default_interface]
+    runtimeclass TimePickerView : Windows.UI.Xaml.Controls.TimePicker {
```

Comment:
FYI once we'll start making modules work with WinUI 3 we'll need to change these things to inherit from `XAML_NAMESPACE.Controls.TimePicker` etc.